doc/drawingElement.md

```diff
@@ -0,0 +1,1186 @@
+
+# Elements to setup canvas (drawing space)
+
+## `pl-drawing` element
+
+A `pl-drawing` element displays a canvas, which can be used to display initial drawing objects or expect drawing objects for grading.
+
+The system of coordinates of the canvas is located at the top/left corner.
+
+#### Sample Element
+
+```html
+<pl-drawing answers-name="fbd">
+</pl-drawing>
+```
+<img src="elements/pl-drawing.png" width=70%>
+
+#### Customizations
+
+Attribute | Type | Default | Description
+--- | --- | --- | ---
+`preview` | boolean | false | `preview = true` is used when canvas is used for display only. `preview = false` expects objects to be placed in the canvas for grading.
+`answers-name` | string | — | Variable name to store data in. This variable is required when `preview = false`.
+`width` | integer | 580 | Horizontal width of the canvas.
+`height` | integer | 320 | Vertical height of the canvas.
+`grid-size` | integer | 20 | Size of the square grid for the canvas background. If `grid-size = 0`, then the background is white.
+`snap-to-grid` | boolen | true | Objects placed in the canvas snap to closest grid point.
+`correct-answer` | string | special | Correct answer for grading. Defaults to `data["correct-answers"][answers-name]`.
+`tol` | float | `0.5*grid-size` | Tolerance to check the position of objects. The error is calculated as the absolute difference between expected position and submitted one.
+`angle-tol` | float | 10 | Tolerance to check angles (in degrees). The error is calculated as the absolute difference between expected angle and submitted one.
+`show-tolerance-hint` | boolean | true | Show tolerance hint under the canvas. The default is `true` when `preview = false`
+`tolerance-hint` | text | "The expected tolerance is 1/2 square grid." | Hint explaining tolerance used when grading objects.
+
+
```

Comment:
Many entries are missing:

````
# Example Implementations

- [link to example question in exampleCourse]().

# See also
- related elements
````

I would say the most helpful is link to fill out is the example implementation in a working question.

Reply:
I added links to the documentation. If possible, other improvements to the documentation could be done later.